app/config.rb

```diff
@@ -41,7 +48,8 @@
         client_options: {
           identifier: oidc_client_id,
           secret: oidc_client_secret,
-          redirect_uri: "#{ENV["PUBLIC_URL"]}/auth/openid_connect/callback"
+          redirect_uri: "#{AppConfig[:frontend_proxy_url]}auth/openid_connect/callback",
+          end_session_endpoint: "#{oidc_end_session_endpoint}?#{AppConfig[:frontend_proxy_url]}"
```

Comment:
There's an issue (already discussed with you) with mlibrary/aspace-oauth/oidc that is preventing this from working fully. Looks like aspace-oauth expects the provider symbol in the route, but we're using `oidc_logout`.

Reply:
Fixed.